<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorControlWidget</class>
 <widget class="QWidget" name="MotorControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление двигателем</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Управление тормозом</string>
   </property>
   <widget class="QLabel" name="dacLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Напряжение: 0.0 В</string>
    </property>
   </widget>
   <widget class="QSlider" name="dacSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="dacZeroBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ноль</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="dacLcd">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
